### ISSUE TITLE
Different transformations per target

### ISSUE THREAD

**Issue by maggo** [remy/bind.js#33]
Hi, I have a config like this

```
bind({
  date: null,
}, {
  date: 'input[name=date],div#display__date',
});
```

and would like to apply different transformation functions to the input and div. So that the value of the input is still valid, but in the div it gets displayed as a human readable string.

Is this possible with the current implementation?

**Comment by remy** [comment]
I can't actually remember myself, but what's the value of `this` inside your transform function? It might give you what you need (otherwise, I think passing in the element to the transform is a good move).

**Comment by maggo** [comment]
In my case I'm using arrow functions so `this` is undefined or rather the current context. If I use a anonymous function it's an object with the `safe` function, so no luck there

**Comment by remy** [comment]
Yeah, I just spotted that in the docs (which I should have really known!). I think a PR that has the transform function (and perhaps the other functions) take the current element as the last argument would be perfect.
